python/tvm/topi/hexagon/qnn/adaptive_avg_pool1d.py

```diff
@@ -1,120 +1,120 @@
-# Licensed to the Apache Software Foundation (ASF) under one

```

Comment:
Not sure what happens in these files. Seems no changes at all

Reply:
Yep idk why git thinks it's different...I'm going to revert this file anyways

Reply:
I got them reverted and will submit a separate PR